Library/Formula/spaceman-diff.rb

```diff
@@ -0,0 +1,39 @@
+class SpacemanDiff < Formula
+  desc "Diff images from the command-line"
+  homepage "https://github.com/holman/spaceman-diff"
+  url "https://github.com/holman/spaceman-diff/archive/v1.0.1.tar.gz"
+  sha256 "98c9420eb769914837e97f4f369523b42c6eb8141175f97cbd7cadf2d3609ede"
+
+  head "https://github.com/holman/spaceman-diff.git"
+
+  depends_on "jp2a"
+  depends_on "imagemagick"
+
+  bottle :unneeded
+
+  def install
+    bin.install "spaceman-diff"
+  end
+
+  def caveats; <<-EOS.undent
+    To finish the installation, create ~/.config/git/attributes and add:
+      *.png  diff=spaceman-diff
+      *.jpg  diff=spaceman-diff
+      *.jpeg diff=spaceman-diff
+      *.gif  diff=spaceman-diff
+
+    Then edit ~/.gitconfig and add:
+
+      [diff "spaceman-diff"]
+        command = #{opt_bin}/bin/spaceman-diff
+    EOS
```

Comment:
Hi @mikemcquaid, I'm not sure what you mean here. In order to get `spaceman-diff` to work with git, the lines in the `caveats` block need to be added to the respective files. Is there a better place to let the user know that they need to do additional work to get `spaceman-diff` to work?